src/commons/RestAPI.js

```diff
@@ -5,13 +5,39 @@ import axios from 'axios';
 import { geometryDAGMA } from '../assets/mockups/summaryData';
 
 class RestAPI {
+
+  /** **************** */
+  /** HOME INFORMATION */
+  /** **************** */
+
+  /**
+   * Request anh areas list
+   */
+  static requestANHAreas() {
+    return RestAPI.makeGetRequest('anh_areas');
+  }
+
+  /**
+   * Request anh area from list
+   * @param name from anh areas list to be searched
+   */
+  static requestAreaSelected(name) {
+    return RestAPI.makeGetRequest(`anh_areas/${name}`);
+  }
+
+  /**
+   * Request sedimentary basins list
+   */
+  static requestSedimentaryBasins() {
+    return RestAPI.makeGetRequest('sedimentary_basins');
+  }
+
   /** *************** */
   /** MAPS IN SUMMARY */
   /** *************** */
 
   /**
    * Request area geometry by id
-   *
```

Comment:
Th line between function description and parameters should be kept. The others i asked you to delete were at the end of documentation (after params or after description with no params)